atomicwrites/__init__.py

```diff
@@ -20,6 +20,8 @@ def _path_to_unicode(x):
 
 if sys.platform != 'win32':
     def _replace_atomic(src, dst):
+        if os.path.exists(dst):
+            os.chmod(src, os.stat(dst).st_mode)
```

Comment:
Actually, I did this deliberately.
I called `os.chmod` in my code, where I use atomicwrite, but some applications,
look at file `mtime` (in my case it is apache httpd rewrite map) and cache the result, but `os.chmod` does not change the mtime, so I had to update `mtime` as well.